main-app/app/tests/core/authenticate/bingo-authentication-proxy.test.js

```diff
@@ -0,0 +1,43 @@
+(function () {
+    'use strict';
+    describe('Test the API wrapper service', function () {
+        var $httpBackend,
+            proxy;
+
+        beforeEach(function () {
+            module('Tombola.BingoClient', function($provide){
+                $provide.value(mocks.bingoApiProxy, 'call');
```

Comment:
You should review the unit testing notes. These `$provide` calls aren't doing what they are supposed to you are probably using the actual services. The code should be like:
````
$provide.value('nameOfThingToReplace', mockToReplaceItWith);
````